helix-view/src/document.rs

```diff
@@ -258,10 +583,11 @@ impl Document {
 
     fn detect_language(&mut self) {
         if let Some(path) = self.path() {
-            let loader = LOADER.get().unwrap();
-            let language_config = loader.language_config_for_file_name(path);
-            let scopes = loader.scopes();
-            self.set_language(language_config, scopes);
+            if let Some(loader) = LOADER.get() {
```

Comment:
This unwrap is always safe because if a LOADER isn't present at application start then things have gone bad.

Reply:
Yeah, it's only there for now because it was necessary to get the benchmarks rolling. I'll switch it back later when I come up with a better solution.